README.md

```diff
@@ -23,6 +23,8 @@ Now you can run the script by typing (You will need netcat installed on the comp
 
     ./init_and_wait_for_db.sh
 
+**You will need to modify config.example.py with the correct database parameters.**
```

Comment:
You shouldn't have to do this, no, since config.example.py fetches the correct database parameters from the environment by default.

This should 'just work' unless we missed some config.